src/modules/count.test.js

```diff
@@ -0,0 +1,55 @@
+/** @jest-environment jsdom */
+import showsCount from './homepage';
```

Comment:
- [ ] Your tests are not passing as they have an error. please kindly ensure they pass and add instructions on how to run your tests in your readme file.